plugins/portals-list.css

```diff
@@ -56,7 +56,7 @@
 
 #portalslist .filters {
   display: grid;
-  grid-template-columns: 1fr auto 1fr auto 1fr auto;
```

Comment:
any reason to go from 3 to 2 columns ?

Reply:
Visuals - 4 factions (n,e,r,m) won't fit into 3 columns 😉
Maybe we can instead add one more to have only 2 rows there, one with all 4 factions and second with the scouting...
And 2 columns only for mobile